### PR TITLE
added permissions note

### PR DESCRIPTION
**Pull request by filipw**
just a footnote, ran into this on a Mac

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,5 @@ Currently some of the funtionality is dependent upon the kind people at adobe ac
 1. clone the latest version of brackets from this branch https://github.com/mat-mcloughlin/brackets/tree/SpecialEdition
 2. Follow the first 6 points in the instructions here https://github.com/adobe/brackets/wiki/How-to-Hack-on-Brackets#setting-up-your-dev-environment
 3. Within brackets click File -> Extension Manager... -> Install from URL... and enter this url https://github.com/OmniSharp/omnisharp-brackets
+
+**Note:** If you are on a non-Windows platform, and notice permission issues in the Brackets Debug Console that prevent Omnisharp from starting (such as `Error: spawn EACCES` etc),you need to ensure the permissions on the Brackets extension are in proper order. To do that, execute `chmod -R a+x /Users/<your username>/Library/Application Support/Brackets/extensions/user/mat-mcloughlin.omnisharp` in your terminal.
```
Comment (nosami) [inline README.md]: ~/Library............. is better?

**Comment by filipw** [comment]
@nosami you are right indeed

**Comment by Yantrio** [comment]
This is incorrect for linux, and users that didnt install the application via the url, but by the package or a direct clone into the repo,
can we add the linux location "~/.config/brackets/extensions/user" please.

maybe we could move towards an FAQs section in the future :)

**Comment by filipw** [comment]
you realize it would have taken less time to merge it and click edit on the readme and change it then? :P

**Comment by Yantrio** [comment]
Done, cheers :)

**Pull request merged**